app/javascript/spec/select/select.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import FormRenderer from '@data-driven-forms/react-form-renderer';
+import { FormTemplate, componentMapper } from '@data-driven-forms/pf3-component-mapper';
+import Select from '../../components/select';
+
+const RendererWrapper = ({ onChange }) => (
+  <FormRenderer
+    onSubmit={ () => {} }
+    FormTemplate={ FormTemplate }
+    componentMapper={{
+      ...componentMapper,
+      'select': Select,
+    }}
+    schema={{
+      fields: [{
+        component: 'select',
+        name: 'selectField',
+        label: 'Choose',
+        options: [
+          {
+            "label": "Dogs",
+            "value": "1"
+          },
+          {
+            "label": "Cats",
+            "value": "2"
+          },
+        ],
+        onChange,
+      }],
+    }}
+  />
+);
+
+describe('Select component', () => {
+  it('should match the snapshot', () => {
+    const wrapper = mount(<RendererWrapper/>);
```

Comment:
I guess it's enough to use `shallow` instead of `mount`, it will make the test less prone to implementation changes coming from the DDF library.

Reply:
I tried it, but got errors.